app/src/full/kotlin/io/homeassistant/companion/android/settings/wear/views/SettingsWearOnboardingView.kt

```diff
@@ -0,0 +1,131 @@
+package io.homeassistant.companion.android.settings.wear.views
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.PreviewLightDark
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import io.homeassistant.companion.android.common.R as commonR
+import io.homeassistant.companion.android.settings.wear.SettingsWearViewModel
+import io.homeassistant.companion.android.util.compose.HomeAssistantAppTheme
+
+@Composable
+internal fun SettingsWearOnboardingView(
+    settingsWearViewModel: SettingsWearViewModel,
+    onInstallOnWearDeviceClicked: () -> Unit,
+    onFinishInstallOnDevices: () -> Unit,
+    onBackClicked: () -> Unit,
+) {
+    val uiState by settingsWearViewModel.settingsWearOnboardingViewUiState.collectAsStateWithLifecycle(
+        SettingsWearViewModel.SettingsWearOnboardingViewUiState(),
+    )
+
+    if (uiState.installedOnDevices) {
+        onFinishInstallOnDevices()
+    }
+
+    SettingsWearOnboardingViewContent(
+        infoTextTitleResource = uiState.infoTextResourceId,
+        shouldDisplayRemoteAppInstallButton = uiState.shouldShowRemoteInstallButton,
+        onInstallOnWearDeviceClicked = onInstallOnWearDeviceClicked,
+        onBackClicked = onBackClicked,
+    )
+}
+
+@Composable
+internal fun SettingsWearOnboardingViewContent(
+    infoTextTitleResource: Int,
+    shouldDisplayRemoteAppInstallButton: Boolean,
+    onInstallOnWearDeviceClicked: () -> Unit,
+    onBackClicked: () -> Unit,
+) {
+    Scaffold(
+        topBar = {
+            SettingsWearTopAppBar(
+                title = {
+                    Text(
+                        stringResource(commonR.string.wear_os_settings_title),
+                        fontWeight = FontWeight.Bold,
+                    )
+                },
+                onBackClicked = onBackClicked,
+                docsLink = WEAR_DOCS_LINK,
+            )
+        },
+    ) { contentPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(contentPadding),
```

Comment:
```suggestion
                .padding(safeBottomPaddingValues())
                .padding(contentPadding),
```
Otherwise the content is not properly display depending on where is the cutout